engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details-info.controller.js

```diff
@@ -61,7 +61,7 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
                     return false;
                 });
 
-                results.unshift({id: null});
+                results.unshift({id: null, name:''});
```

Comment:
The spacing looks a little funny here, eslint may or may not complain (its a really old version with bastion_katello files), could you do `name: ''`?

Reply:
oh lemme do that

Reply:
I had to add it because the repo edit page showed "Undefined" as the first item. Needed it  to be blank

Reply:
updated